sorter.lic

```diff
@@ -0,0 +1,179 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_development#sorter
+=end
+
+custom_require.call(%w[common])
+
+@settings = get_settings
+@settings.sorter['ignore_categories'] ||= "nil"
```

Comment:
Why the string nil and not actual nil?

Also since lich overrides nil, this isn't necessary - if you try to lookup a key that doesn't exist, you get `nil` back
```
>;e a_hash = {}; echo a_hash['ignore_categories']        
--- Lich: exec1 active.                                  
[exec1: ]                                                
```

Reply:
Gave me an error if ignore_categories was blank. Using just nil also gave an error, a string "nil" was easier fix.

Reply:
Ok, are you the author of the sorter script?